src/services/goalServices/getGoalsByActivityRecipient.test.js

```diff
@@ -793,6 +793,38 @@ describe('Goals by Recipient Test', () => {
       expect(goalRowsx.length).toBe(3);
     });
 
+    it('Retrieves and sorts for merged goals by Recipient', async () => {
+      const { count, goalRows } = await getGoalsByActivityRecipient(recipient3.id, 1, {
+        sortBy: 'mergedGoals', sortDir: 'desc', offset: 0, limit: 20, goalIds: [goalIds[9]],
+      });
+      const countx = count;
+      const goalRowsx = goalRows;
+      expect(countx).toBe(3);
+      expect(goalRowsx.length).toBe(3);
+      expect(goalRowsx[0].id).toBe(goalIds[9]);
```

Comment:
Should we check second and third position of resulting sorts?

Reply:
sure